### ISSUE TITLE
Adjust README.md to place PyCNAL_legacy among other projects?

### ISSUE THREAD

**Issue by drf5n** [ESMG/PyCNAL_legacy#20]
Could you edit README.md to point towards the canonical projects this supercedes and may be superceded by? (if it is ready to supercede them yet?) 

It is difficult to follow all the ROMS grid generation projects from easygrid, seagrid, pyroms, Octant, pyroms, PyCNAL_legacy, and maybe PyCNAL.  If the ones in current development pointed at each other, it might help folks figure out which ones to try and how to use them best.

* https://www.myroms.org/wiki/easygrid
* https://www.myroms.org/wiki/seagrid https://github.com/sea-mat/seagrid
* https://github.com/hetland/octant
* https://github.com/ESMG/pyroms
* https://github.com/hetland/pygridgen
* https://github.com/ESMG/PyCNAL_legacy
* https://github.com/ESMG/PyCNAL

**Comment by kshedstrom** [comment]
Good point. In particular, I think PyCNAL_legacy can be made to go away now that pyroms is version 3 compliant.
